.babelrc

```diff
@@ -5,8 +5,10 @@
     "@babel/preset-flow"
   ],
   "plugins": [
+    ["@babel/plugin-transform-runtime"],
```

Comment:
We could also use `babel-preset` as it loads the appropriate transformers based on the target environment.